### ISSUE TITLE
CIFAR-100 HP

### ISSUE THREAD

**Issue by RongKaiWeskerMA** [SHI-Labs/Compact-Transformers#21]
Thanks for this interesting work! I am just wondering if you can provide the command line for training the cifar-100 dataset?

**Comment by alihassanijr** [comment]
Thank you for your interest.
We normally used the same hyperparameters as CIFAR-100. This means that the following is a good place to start:
```
python main.py \
       --dataset cifar100 \
       --model cct_7 \
       --conv-size 3 \
       --conv-layers 1 \
       --warmup 10 \
       --batch-size 128 \
       --checkpoint-path /path/to/checkpoint.pth \
       path/to/cifar100
```
More details on our newest results are in the latest arXiv preprint, as we train these models for longer and with a different weight decay.
If you have any questions, please do not hesitate to ask.

**Comment by alihassanijr** [comment]
I'm closing the issue now, if you still have questions feel free to open it back up.
